frontend/src/utils/operatorDescriptors.js

```diff
@@ -415,7 +415,10 @@ const operatorFieldValidators = {
       required: true
     },
     description: {
-      required: true
+      description: {
+        // please note, that more detailed validation is done inside of DescriptionEditor
+        validator: text => (/(^# [^\r?\n]+)/m.test(text) ? true : null)
```

Comment:
If the test fails it should return a string for the error message, if it passes return null.